content/docs/user-guide/how-to/best-practices.md

```diff
@@ -0,0 +1,43 @@
+# Best Practices for DVC Projects
+
+Data scientists, engineers, or managers may already know or can easily find
+answers to some of these questions. However, the variety of answers and
+approaches makes data science collaboration a nightmare. **A systematic approach
+is required.**
+
+## Questions on...
+
+### Source code and data versioning
+
+- How do you avoid discrepancies between
```

Comment:
@imhardikj this is a WIP begining to address #72 (listed in #899). May be a good first one to try and finish along with this PR.